app-doc/hplip/autobuild/postinst

```diff
@@ -1,2 +1,4 @@
 # Avoid hp-check error.
-systemctl enable cups --now
+if ! systemd-detect-virt --container; then
```

Comment:
Insufficient for chroot detection, see 5fcb1b25731d98b21ac1948b24c2cfbd6ce8d8b5.